integration/rest/src/main/java/it/pagopa/selfcare/dashboard/connector/rest/config/PartyManagementRestClientConfig.java

```diff
@@ -0,0 +1,16 @@
+package it.pagopa.selfcare.dashboard.connector.rest.config;
+
+import it.pagopa.selfcare.commons.connector.rest.config.RestClientBaseConfig;
+import it.pagopa.selfcare.dashboard.connector.rest.client.PartyManagementRestClient;
+import it.pagopa.selfcare.dashboard.connector.rest.client.PartyProcessRestClient;
+import org.springframework.cloud.openfeign.EnableFeignClients;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Import;
+import org.springframework.context.annotation.PropertySource;
+
+@Configuration
+@Import(RestClientBaseConfig.class)
+@EnableFeignClients(clients = PartyManagementRestClient.class)
+@PropertySource("classpath:config/party-mgmt-rest-client.properties")
+class PartyManagementRestClientConfig {
```

Comment:
How are you handling the Authentication header forward to invoke REST secured api?